.github/workflows/check-spelling.yml

```
@@ -0,0 +1,16 @@
+name: Check spelling # cSpell:ignore refcache
+
+on:
+  pull_request:
+
+jobs:
+  check-formatting:
```

The job ID should be changed (this is probably a copy-paste error); candidate names: `check-spelling` or just `check` or even maybe just `main`?

```suggestion
  main:
```